stripper/ze_santassination_v3.cfg

```diff
@@ -959,3 +959,520 @@ modify:
 		"OnUser1" "!self,Kill,,0.00,1"
 	}
 }
+
+;Remove annoying stacked sounds
```

Comment:
Maybe pitch this to luff if you think it's necessary, I'd like to refrain from making non-necessary changes while luff is actively maintaining a stripper for the map. Personally I haven't even heard the sounds give any issues, other than when multiple benders are going at the same time on extreme.